Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py

```diff
@@ -545,6 +548,41 @@ def extract_urls_and_emails_from_pdf_file(file_path: str, output_folder: str) ->
     return urls_ec, emails_ec
 
 
+def extract_hash_contexts_from_pdf_file(file_path: str) -> list[dict[str, Any]]:
+    hash_contexts: list[dict[str, Any]] = []
+    # Get hashes from the binary file
+    hashes_in_file = get_hashes_from_file(file_path)
+    for hash_type, hashes in hashes_in_file.items():
+        if hashes:
+            hash_contexts.extend(convert_hash_to_context(hash_type, hashes))
```

Comment:
You can check it before the for loop and then have it a one liner, but its not critical.